A6b (0.06) changes (#32)
CarterKoala/wca-regulations-wrc: wca-regulations.md

- A6) Stopping the solve:
    - A6a) The competitor releases the puzzle before stopping the solve, and stops the solve by stopping the timer. If a stopwatch is in use, the judge stops the stopwatch as soon as the competitor does this.
    - A6b) The competitor is responsible for stopping the Stackmat timer correctly.

>         - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time of 0.05 seconds or lower, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.

Reviewer: This is not correct. A timer showing a time of `0.053` should result in an extra, but that's not covered by this sentence.

>         - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. Exception: No extra attempt is required if the WCA Delegate determines that the timer was stopped deliberately or as a result of the competitor not starting the timer correctly (see [Regulation A4](regulations:regulation:A4)).

Reviewer 2: ```suggestion
        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. Exception: No extra attempt is required if the WCA Delegate determines that the timer was stopped deliberately, or stopped as a result of the competitor not starting the timer correctly (see [Regulation A4](regulations:regulation:A4)).
```

>         - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: The WCA Delegate may provide an extra attempt if they believe the timer may have malfunctioned and they are sure the competitor did not stop the timer deliberately.

Reviewer 2: ```suggestion
        - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: The WCA Delegate may provide an extra attempt if they believe the timer may have malfunctioned and the competitor did not stop the timer deliberately.
```
"they are sure" is a high standard for evidence I don't think we use elsewhere. Whether something happened or not should be up to the Delegate to decide if there isn't conclusive evidence.

    - A6a) The competitor releases the puzzle before stopping the solve, and stops the solve by stopping the timer. If a stopwatch is in use, the judge stops the stopwatch as soon as the competitor does this.
    - A6b) The competitor is responsible for stopping the Stackmat timer correctly.

>         - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.

Reviewer 2: ```suggestion
        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. Exception: No extra attempt is required if the WCA Delegate determines that the timer was stopped deliberately or as a result of the competitor not starting the timer correctly (see [Regulation A4](regulations:regulation:A4)).
```
I think were were also in favor of something like this to allow for more situations in which an extra would not be automatic?
